_boards_cm/45drives-storinator-jr.md

```diff
@@ -1,11 +1,11 @@
 ---
 layout: board
 title: "45Drives' Storinator Jr"
+short_description:
+status: A mini version of the Storinator built with a Raspberry Pi.
```

Comment:
I believe this is supposed to be `short_description`?

Reply:
And the status would be prototype

Reply:
SHJFDJLKDF GOOD CATCH SORRY I'll fix that shortly! Whoops.